### PR TITLE
Improve readability and clarity.

### PR DESCRIPTION
**Pull request by Linicks**

### CODE REVIEW

File: src/doc/trpl/README.md [hunk 1]
```diff
@@ -1,21 +1,25 @@
-% The Rust Programming Language
+% The Rust Programming Language Book
```
Comment (steveklabnik) [inline src/doc/trpl/README.md]: I see what you're going for here, but this isn't the title. "Book" is redundant.

File: src/doc/trpl/README.md [hunk 2]
```diff
@@ -1,21 +1,25 @@
...
+requirements, and low-level code like device drivers and operating systems. A 
+major advantage that Rust has over other programming languages are the built in
+safety features that don't impact performance.  The language, and the compiler 
+are designed to work together providing comprehensive compile-time safety checks
+that produce no runtime overhead, and eliminates common bugs like data races.  Rust
+also provides many ‘zero-cost abstractions’ giving it the look and feel of a 
+high-level language, while maintaining the precise control of a low-level language.  
+Rust is versatile enough to build everything from game engines to RESTful web services.  
+By reading this book, and learning Rust, you will be able to develop higher quality software 
+that you can count on.
```
Comment (steveklabnik) [inline src/doc/trpl/README.md]: Is this extra sentence the only difference here? It's a bit hard to tell, exactly. I think you've added two spaces after some periods? It should be just one space.

Comment (steveklabnik) [inline src/doc/trpl/README.md]: I'm not entirely sure that I want to claim this, at the end, but I also can't quite articulate why.

**Comment by steveklabnik** [comment]
Thank you for the patch, @Linicks ! I'm feeling a bit ambivalent about some of it, let's talk it out?

File: src/doc/trpl/README.md (hunk 2 above)
Comment (nagisa) [inline src/doc/trpl/README.md]: Double spaces is a typesetting-machine-era thing. HTML completely ignores repeating spaces (unless they appear in `<pre>` or in an element with `white-space: pre;` style), so there’s absolutely no reason to have these.

**Comment by steveklabnik** [comment]
Okay @Linicks ! It's been a month and I haven't heard from you. Let me know if you're interested in updating this PR.

**Pull request closed without merging**